plugins/woocommerce-admin/client/marketplace/components/my-subscriptions/my-subscriptions.tsx

```diff
@@ -16,10 +20,7 @@ import {
 	AvailableSubscriptionsTable,
 	InstalledSubscriptionsTable,
 } from './table/table';
-import {
-	availableSubscriptionRow,
-	installedSubscriptionRow,
-} from './table/table-rows';
+import { subscriptionRow } from './table/table-rows';
```

Comment:
Thanks for this refactor! And thanks for separating it into commits, made my review easy :)